### PR TITLE
Adds tests for String Keys. 

### PR DESCRIPTION
**Pull request by hemanik**
Units Tests for String Keys implementation as per the [README](https://github.com/arquillian/arquillian-reporter#string-keys) have been included.

### CODE REVIEW

**Comment by MatousJobanek** [comment]
please clean the commit history - to contain only relevant commits

File: core/reporter-impl/src/test/java/org/arquillian/reporter/impl/base/AbstractReporterTestBase.java [hunk 1]
```diff
@@ -102,6 +103,7 @@ private void registerBuilders() {
     private void initiateStringKeys() {
         ArrayList<StringKey> stringKeys = new ArrayList<>();
         stringKeys.add(new ReporterCoreKey());
+        stringKeys.add(new TestExtensionStringKey());
```
Comment (MatousJobanek) [inline core/reporter-impl/src/test/java/org/arquillian/reporter/impl/base/AbstractReporterTestBase.java]: This should be placed here: https://github.com/arquillian/arquillian-reporter/pull/18/files#diff-201783225109103171e4bf201ba27b33R76

File: core/reporter-impl/src/test/java/org/arquillian/reporter/impl/model/StringKeysTest.java [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
+
+    @Test
+    public void should_load_value_ignoring_leading_whitespaces_from_properties_file() {
+        String value = DRONE_REPORT_SCOPE.getValue();
+
+        assertThat(value).isEqualTo("Scope");
+    }
+
+    @Override
+    protected void addAdditionalExtensions(List<Class<?>> extensions) {
```
Comment (MatousJobanek) [inline core/reporter-impl/src/test/java/org/arquillian/reporter/impl/model/StringKeysTest.java]: If there is no implementation then there is no need to keep it here...

File: core/reporter-impl/src/test/java/org/arquillian/reporter/impl/model/StringKeysTest.java [hunk 3]
```diff
@@ -1,7 +1,83 @@
...
+
+    }
+
+    @Override
+    protected void addReporterStringKeys(List<StringKey> stringKeys) {
+
+    }
+
+    @Override
+    protected void registerBuilders(BuilderRegistryDelegate builderRegistry) {
```
Comment (MatousJobanek) [inline core/reporter-impl/src/test/java/org/arquillian/reporter/impl/model/StringKeysTest.java]: same here - if there is no implementation then there is no need to keep it here...

**Comment by hemanik** [comment]
Opened [new pull request](https://github.com/arquillian/arquillian-reporter/pull/19) to do away with untidy commit history.

**Pull request closed without merging**